docs/content/contributing/internal/RemoteCache.md

```diff
@@ -0,0 +1,42 @@
+# Remote Cache
+
+???+ warning 
+    Currently available only for Avito employees because accessible only via corporate network
+
+???+ warning
+    Persistence volume not set for now, deleting a pod will lead to losing cache.
+
+Official docs: https://docs.gradle.com/build-cache-node
+
+## How to enable locally
+
+???+ Prerequisites 
+    Ask for <GRADLE_CACHE_NODE_HOST> in internal chat
+    
+Set environment variable `GRADLE_CACHE_NODE_HOST`
```

Comment:
Why not project property? It's used only in Gradle process and can be hidden from others.

Reply:
I see, it's also used in deploy